patch.cabal

```diff
@@ -33,12 +33,13 @@ library
   build-depends: base >= 4.9 && < 4.14
                , constraints-extras >= 0.3 && < 0.4
                , containers >= 0.6 && < 0.7
-               , dependent-map >= 0.3 && < 0.4
-               , dependent-sum >= 0.6 && < 0.7
+               , dependent-map >= 0.3 && < 0.5
+               , dependent-sum >= 0.6 && < 0.8
                , lens >= 4.7 && < 5
                , semigroupoids >= 4.0 && < 6
                , transformers >= 0.5.6.0 && < 0.6
-               , witherable >= 0.3 && < 0.3.2
+               , witherable >= 0.3 && < 0.4
+               , some >= 1 && < 1.1
```

Comment:
Please sort.

Reply:
Uh, I am sorry. I only added the `some` dependency in the hope, that it might fix the issue. 
I don't have a ghc 8.0.2 handy, so I can only test it by uploading into the PR.

But apparently the import of `some` didn’t fix the issue. So I need to keep looking.